apis/v1alpha1/opampbridge_capabilities.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+type (
+	// OpAMPBridgeCapability represents capability supported by OpAMP Bridge.
+	// +kubebuilder:validation:Enum=AcceptsRemoteConfig;ReportsEffectiveConfig;ReportsOwnTraces;ReportsOwnMetrics;ReportsOwnLogs;AcceptsOpAMPConnectionSettings;AcceptsOtherConnectionSettings;AcceptsRestartCommand;ReportsHealth;ReportsRemoteConfig
+	OpAMPBridgeCapability string
```

Comment:
Should we instead import these directly from the [opamp code](https://github.com/open-telemetry/opamp-go/blob/main/protobufs/opamp.pb.go)? I think the answer is _probably no_ but just a thought...

Reply:
In [opamp code](https://github.com/open-telemetry/opamp-go/blob/main/protobufs/opamp.pb.go), the underlying type for `AgentCapabilities` is `int32`, on the other hand, CRD expects `string`. So, I don't think we can directly use that.